app/github/views.py

```diff
@@ -105,6 +112,14 @@ def github_logout(request):
         request.session.pop('access_token_last_validated')
         Profile.objects.filter(handle=handle).update(github_access_token='')
 
+        # record a useraction for this
+        if Profile.objects.filter(handle=handle).count():
```

Comment:
We could modify this to `get` the specific profile first so we only perform one query versus the second below.

Reply:
yes... the reason i didnt do that is because i was worried about generating an exception if for some reason that handle didnt exist.